File: programmers/난이도별/level01.수박수박수박수박수박수?/Suhyeon.java
class Suhyeon {
    public String Suhyeon(int n) {
        String answer = "";
        
        for(int i=0;i<n;i++){
            if(i%2==0) answer+="수";
            else answer+="박";
        }
        
        return answer;
    }
}

File: programmers/난이도별/level01.문자열_내_p와_y의_개수/suhyeon.js
function suhyeon(s){   
    var pExp = /p/gi;
    var yExp = /y/gi;
    
    var pResult = s.match(pExp);
    var yResult = s.match(yExp);
    
    if(pResult.length == yResult.length){
        return true;
    }
    return false;
}
